src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -195,7 +194,12 @@ where
                     panic_message::panic_message(&payload).to_owned(),
                 ))
             })
-            .inspect_err(|error| error!(%error, "error when process message"));
+            .inspect_err(|error| {
+                error!(
+                    error = error as &dyn std::error::Error,
+                    "error when process message"
+                )
+            });
```

Comment:
why to change this

Reply:
Previously we used `Display`(`%`). By directly passing `&dyn Error`, the sources are also getting printed:

https://docs.rs/tracing-subscriber/latest/src/tracing_subscriber/fmt/format/mod.rs.html#1234

However it's ugly to cast to `&dyn` manually and I'm not quite sure why coercion does not work here. Need further investigation. 🙁